tabpy/models/scripts/SentimentAnalysis.py

```diff
@@ -3,8 +3,16 @@
 from nltk.sentiment.vader import SentimentIntensityAnalyzer
 import sys
 from pathlib import Path
-sys.path.append(str(Path(__file__).resolve().parent.parent.parent / 'models'))
-from utils import setup_utils
+from tabpy.models.utils import setup_utils
+
+
+import ssl
+try:
+    _ctx = ssl._create_unverified_context
+except AttrubuteError:
```

Comment:
`AttributeError`

Reply:
I assume this code path hasn't been tested

Reply:
No, it hasn't. Removing try/catch - it better fail with exception IMO.